File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/A. 1 to N/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemA
    {
        static void Main(string[] args)
        {
            ushort x = ushort.Parse(Console.ReadLine());
            ushort i = 1;
            loop:
                Console.WriteLine(i++);
                if(i <= x) goto loop;
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/A. 1 to N/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short x, i = 1;
    cin >> x;
    loop:
        cout << i++ << endl;
        if(i <= x) goto loop;
}